packages/VSNumberCompare/index.js

```diff
@@ -129,19 +62,107 @@ function buildLegend(values, contrast, styles) {
       commonStyles.push(styles.contrast);
     }
 
-    legend.push(<Text key={ `${ value.unit }` } style={ [ ...commonStyles ] }>{ value.unit.toUpperCase() }</Text>);
-    legend.push(<Text key={ `${ value.unit }:` } style={ [ ...commonStyles, styles.legendSpacing ] }>{ ':' }</Text>);
+    legend.push(<Text key={ `${ value.unit }` } style={ [ ...commonStyles ] }>{value.unit.toUpperCase()}</Text>);
+    legend.push(<Text key={ `${ value.unit }:` } style={ [ ...commonStyles, styles.legendSpacing ] }>{':'}</Text>);
 
-    return value; // Make linter happy
+    return value;
   });
 
-  // Remove trailing colon
   legend.pop();
 
   return legend;
 }
 
-VSNumberCompare.propTypes = {
+class UI extends React.Component {
+  static get defaultProps() {
```

Comment:
Why make this a `get`ter rather than just plain object?

Reply:
It could be changed into,

class UI extends React.Component {
static defaultProps = {
title: 'Defaults',
values: [
{
unit: 'defs',
value: 0,
color: 'black',
},
],
legendPosition: 'bottom',
contrast: false,
fontMain: 'TradeGothicBold',
fontSecondary: 'TradeGothicThin',
fontSize: 72,
fontRatio: 4,
fontMinimum: 16,
numberSpacing: 1,
legendSpacing: 2,
}

...

However, while the above using static defaultProps = { ... } is valid, eslint underscores an unexpected token of = and //eslint-disable-line does not remove the flag. When made into a getter it becomes eslint compliant.